main_test.go

```diff
@@ -134,3 +138,43 @@ func initArtifactoryCli() {
 		configArtifactoryCli = createConfigJfrogCLI(cred)
 	}
 }
+
+func testCreateConfFile(dirs []string, resolver, deployer string, t *testing.T, confType artifactoryUtils.ProjectType) error {
```

Comment:
The function name probably does not match its functionality.
I'd also suggest adding a description for this function.

Reply:
testCreateConfFile  -> createConfigFileForTest